tests/test_server.py

```diff
@@ -123,6 +126,48 @@ def test_status_not_ready_agent(rasa_app: SanicTestClient):
     assert response.status == 409
 
 
+@pytest.fixture
+def formbot_data():
+    return dict(
+        domain="examples/formbot/domain.yml",
+        config="examples/formbot/config.yml",
+        stories="examples/formbot/data/stories.md",
+        nlu="examples/formbot/data/nlu.md",
+    )
+
+
+def test_train_status(rasa_server, rasa_app, formbot_data):
+    with ExitStack() as stack:
```

Comment:
why do we need the `ExitStack`? We should re-use the existing read functions from `utils.io`

Reply:
@msamogh ?

Reply:
Used it so that all of them can be loaded together with a single common statement. Do you think it'd still be better to go with `utils.io`?

Reply:
Ah got it now. I'd just call the utils.io function for them cause these functions have the defaults for encoding and so on.